src/app/workflow/workflow-file-editor/workflow-file-editor.component.ts

```diff
@@ -20,12 +20,10 @@ export class WorkflowFileEditorComponent extends FileEditing {
   @Input() descriptorType: string;
   @Input() set selectedVersion(value: WorkflowVersion) {
     this._selectedVersion = value;
+    this.clearSourceFiles();
```

Comment:
@agduncan94 watch out, strictly speaking this is not the same since the clear was happening after the content of the branch and now its happening before